tests-web/www/js/main.js

```diff
@@ -82,42 +77,34 @@ export class StreetExplorer {
 }
 
 class TestCase {
-  constructor(app, name, osmXML, drivingSide, layers, bounds) {
+  constructor(app, name, osmXML, drivingSide, bounds) {
     this.app = app;
     this.name = name;
     this.osmXML = osmXML;
     this.drivingSide = drivingSide;
-    this.layers = layers;
     this.bounds = bounds;
   }
 
-  cleanup() {
-    for (const layer of this.layers) {
-      this.app.removeLayer(layer);
-    }
-  }
-
   static async loadFromServer(app, name) {
     const prefix = `tests/${name}/`;
     const osmInput = await loadFile(prefix + "input.osm");
-    const rawMap = loadFile(prefix + "raw_map.json");
-    const network = loadFile(prefix + "road_network.dot");
+    const rawMap = await loadFile(prefix + "raw_map.json");
+    const network = await loadFile(prefix + "road_network.dot");
 
-    const rawMapLayer = makePlainGeoJsonLayer(await rawMap);
+    const rawMapLayer = makePlainGeoJsonLayer(rawMap);
     const bounds = rawMapLayer.getBounds();
 
-    var layers = [];
-    layers.push(app.addLayer("Geometry", rawMapLayer));
-    layers.push(app.addLayer("OSM", makeOsmLayer(osmInput)));
-    layers.push(
-      app.addLayer("Network", await makeDotLayer(await network, { bounds }))
-    );
+    var group = new LayerGroup("built-in test case", app.map);
+    group.addLayer("Geometry", rawMapLayer);
+    group.addLayer("OSM", makeOsmLayer(osmInput), { enabled: false });
+    group.addLayer("Network", await makeDotLayer(network, { bounds }));
```

Comment:
I want to disable this one by default, but if I try that, then for some reason the SVG shows up covering the entire screen and breaks everything. There's some funniness with `makeDotLayer` that I don't understand yet

Reply:
Oh... Adding it to the map is what moves it out of the DOM i guess... 

The map doesn't make much sense anyway, until it is layed out on an appropriately shaped canvas, and with a sensibly positioned nodes.